integration/token/fungible/views/auditor.go

```diff
@@ -148,3 +149,79 @@ func (p *RegisterAuditorViewFactory) NewView(in []byte) (view.View, error) {
 	f := &RegisterAuditorView{}
 	return f, nil
 }
+
+type CurrentHolding struct {
+	EnrollmentID string `json:"enrollment_id"`
+	TokenType    string `json:"token_type"`
+}
+
+// CurrentHoldingView is used to retrieve the current holding of token type of the passed enrollment id
+type CurrentHoldingView struct {
+	*CurrentHolding
+}
+
+func (r *CurrentHoldingView) Call(context view.Context) (interface{}, error) {
+	w := ttx.MyAuditorWallet(context)
+	assert.NotNil(w, "failed getting default auditor wallet")
+
+	auditor := ttx.NewAuditor(context, w)
+
+	aqe := auditor.NewQueryExecutor()
+	defer aqe.Done()
+
+	filter, err := aqe.Holdings().ByEnrollmentId(r.EnrollmentID).ByType(r.TokenType).Execute()
+	assert.NoError(err, "failed retrieving holding for [%s][%s]", r.EnrollmentID, r.TokenType)
```

Comment:
mh ... Angelo, I must say that I don't like this way of error handling. The call function API allows us to return an error but here we use an assertion ... panic / maybe recover somewhere ... or crash.

Reply:
I understand, indeed developers can handle error as they better prefer. If a panic is generated, the context catches it and send back to the caller of the view the error generated.
We use this approach in all integration tests. 
Can we keep it?

Reply:
ah I didn't recognized that this is in the integration test, therefore yes let's keep it here.

Reply:
okay